src/app/dynamic/enterprise/metering/schedule-config/template.html

```diff
@@ -80,14 +82,31 @@
                fxLayoutAlign=" center">
             <span>Report Scope</span>
             <div class="km-icon-info km-pointer tooltip"
-                 matTooltip="Range of days captured in each report."></div>
+                 matTooltip="Number of days captured in each report."></div>
           </div>
         </th>
         <td mat-cell
             *matCellDef="let element">{{element.interval}}
         </td>
       </ng-container>
 
+      <ng-container matColumnDef="schedule">
```

Comment:
IMHO, instead of duplicating control names wwe can use existing enum Controls defined


e.g Usage

[matColumnDef]="column.<NAME_OF_CONTROL_DEFINED_IN_ENUM>"

Benefits:
- Avoid duplcation
- Avoid Errors while typing
- Single point of change if control name changes from TS file